### DISCUSSION TITLE
OPTIONS OF INPUT FIELD TO ONLY SHOW ITEMS IN RELATION TO THE VALUE IN PREVIOUS INPUT

### DISCUSSION THREAD

**Discussion by farajabien** [Budibase/budibase#8710] [Help] [answered]
I'm working on an app that requires one to pick a country and then a city. How do I make my dropdown options to only show options based on the country selected in the previous input.
So if I pick England as a country, I shouldn’t be able to see cities from France when I'm selecting a City.

Here is an example:
<img width="714" alt="Screenshot 2022-11-17 at 15 44 35" src="https://user-images.githubusercontent.com/43667570/202449897-360255d1-7739-4deb-80a2-6e2c052a63d3.png">

As you can see I have selected `province` as `province 1` and it only has 2 districts `district1` and `district2`

**How do I make the input dynamic in that you are only allowed to see districts from the chosen province as options???**

**Comment by melohagan** [comment]
Hey @farajabien 

> So if I pick England as a country, I shouldn’t be able to see cities from France when I'm selecting a City.

This functionality will need filtering added to a relationship picker. Work on that was started here: https://github.com/Budibase/budibase/pull/8897

This may help as well: https://docs.budibase.com/docs/filter-by-relationships

**Reply by holgerbermeo** [reply]
It doesn't solve the problem, I'm wanting to do something similar. What we would like is in a form, to be able to choose from a list options that are filtered by another table.

![image](https://user-images.githubusercontent.com/59984753/225937456-e9eabe23-a3a9-4e58-aa88-5d005dd209b8.png)

For example, in a form to register "intervencionvial", I would like to choose (1)"parroquias" that only belong to one (2)"canton" and not for the entire list to appear
![image](https://user-images.githubusercontent.com/59984753/225938338-afb20496-32c2-412d-a82a-42e8ef02d8bd.png)

![image](https://user-images.githubusercontent.com/59984753/225938617-2696b8e8-1df8-41fb-a22c-93325a3c3d9b.png)

![image](https://user-images.githubusercontent.com/59984753/225938725-47649de3-e5c4-48fd-aa24-b3e6f74cd210.png)

For example, that column should filter the options in (1) "Parroquias" once I have chosen a (2) "Canton"

**Reply by holgerbermeo** [reply]
Another more visual example. I have some data A-B-X and on the other hand I have 1-2-3-5-6-7-8-9-10, but 1-2-3 is inside A and 5-6-7 is within B and 8-9-10 within X. If in this case I choose option B in my first question, I only want the data related to B to appear, which are 5-6-7

![image](https://user-images.githubusercontent.com/59984753/225961578-36625336-695b-42ee-b67a-e585e0be6a49.png)

![image](https://user-images.githubusercontent.com/59984753/225961069-28afaf25-4077-42a4-88b3-6cd904ef0114.png)

**Comment by melohagan** [comment, marked as answer]
Hey @farajabien @holgerbermeo 

Just to let you guys know, you can now filter relationship pickers which should solve for this use-case!

There is a recipe tutorial you can look at here: https://docs.budibase.com/docs/cascading-dropdown-filters
